calico-enterprise_versioned_docs/version-3.14/networking/egress/egress-gateway-maintenance.mdx

```diff
@@ -48,25 +48,30 @@ Before we can observe these annotations, we must first configure a _termination
 
 In order to widen our maintenance window, we must adjust the `terminationGracePeriodSeconds` field on our egress gateway pods. The amount of time we allot for the termination grace period will dictate how much time a dependent workload has to prepare for a gateway going down.
 
-Let's add a termination grace period of 60 seconds to all pods in our egress gateway deploymment, so that our egress-dependent workloads have a wider window to react:
+Let's add a termination grace period of 60 seconds to all pods in our egress gateway deploymment, so that our egress-dependent workloads have a wider window to react. The Egress Gateway resource must be patched with the new `terminationGracePeriodSeconds` as below:
```

Comment:
```suggestion
Let's add a termination grace period of 60 seconds to all pods in our egress gateway deploymment, so that our egress-dependent workloads has more time to react. The Egress Gateway resource must be updated with the new `terminationGracePeriodSeconds` as below:
```

Reply:
Done